templates/remix/package.json

```
@@ -23,6 +23,7 @@
     "@types/react": "^18.2.20",
     "@types/react-dom": "^18.2.7",
     "@typescript-eslint/eslint-plugin": "^6.7.4",
+    "@typescript-eslint/parser": "^7.0.1",
```

I think we should sync this with the semver of the plugin itself - the carat should ensure it gets the latest within the `6.x` semver release

```suggestion
    "@typescript-eslint/parser": "^6.7.4",
```